src/components/DeleteMember/DeleteMember.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { PureComponent } from 'react';
+import { graphql } from 'react-apollo';
+import Button from 'material-ui/Button';
+import Dialog from 'material-ui/Dialog';
+import DialogActions from 'material-ui/Dialog/DialogActions';
+import DialogContent from 'material-ui/Dialog/DialogContent';
+import DialogContentText from 'material-ui/Dialog/DialogContentText';
+import DialogTitle from 'material-ui/Dialog/DialogTitle';
+
+import DeleteMemberMutation from './DeleteMemberMutation';
+
+class DeleteMember extends PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      open: false
+    };
+  }
+
+  handleOnClick = bool => {
+    this.setState({ open: bool });
+  };
+
+  submitMutation = () => {
+    const { user, mutate, refetch, closeDialog } = this.props;
+    mutate({
+      variables: { id: parseInt(user.id, 0) }
+    }).then(({ data }) => {
+      closeDialog();
+      refetch();
+    });
+  };
+
+  render() {
+    const { open } = this.state;
+    return (
+      <div>
+        <Button raised onClick={() => this.handleOnClick(true)}>
+          Delete
+        </Button>
+        <Dialog open={open} onRequestClose={() => this.handleOnClick(false)}>
+          <DialogTitle>Are you sure you want to delete member?</DialogTitle>
+          <DialogContent>
+            <DialogContentText>
+              This action can't be reverted.
+            </DialogContentText>
+          </DialogContent>
+          <DialogActions>
+            <Button raised onClick={() => this.submitMutation()}>
+              Confirm
+            </Button>
+            <Button raised onClick={() => this.handleOnClick(false)}>
+              Cancel
+            </Button>
+          </DialogActions>
+        </Dialog>
```

Comment:
A thought I had this morning:

We might want to put this "Confirmation Dialog" idea into it's own component. It's a theme we will reuse to delete other objects or to perform 1 time actions that cannot be undone (not always a delete).

Reply:
Yeah. I'll try and clean this up tonight. Probably gonna include that tree shaking thing.

Reply:
Double check that it's not already included. It might be already setup in `create-react-app`. I'm pretty sure it uses Webpack 3 already.

Reply:
Update: It totally does already use it so we're good!

https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/package.json